gnd/src/main/java/com/google/android/gnd/repository/local/EditType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.repository.local;
+
+public enum EditType {
+  UNKNOWN(0),
+  CREATE(1),
+  UPDATE(2),
+  DELETE(3);
+
+  public final int intValue;
```

Comment:
Can this be package-private? That way we enforce using the Converters class to convert (and avoid some of the brittleness risk that comes with public fields).

(I personally would prefer a getter instead of ever exposing the field directly so we have an abstraction that gives flexibility to change how the value is implemented, but those risks would also be lower if this is package-private instead of public.)